week1/project/EcommerceVika/ecommerce-shop-project/src/components/FilteredProductsByCategory.js

```diff
@@ -0,0 +1,9 @@
+
+  const FilteredProductsByCategory = (products, category) => {
+  return category ? products.filter(
+        (product) => product.category === category.split(":")[1].trim()
+      )
+    : products;
+ };
+  
+export default FilteredProductsByCategory;
```

Comment:
as I told in the previous comment this is a normal function not a component

Reply:
I will make changes this night according to your comments. Thank you